### ISSUE TITLE
Creating React App from scratch documentation missing

### ISSUE THREAD

**Issue by Zauberfisch** [reactjs/reactjs.org#3183]
Currently, the getting started docs have this line:
```
If you prefer to set up your own JavaScript toolchain from scratch, check out this guide that re-creates some of the Create React App functionality.
``` 
which links to this article: https://blog.usejournal.com/creating-a-react-app-from-scratch-f3c693b84658
However, the code samples of the article are now going 404. 

I believe that it is very important to provide people an option to have their own, minimalistic toolchain. (or rather the documentation for it). Especially for beginners. 
I know there is the `eject` command for the create react app utility, but that includes a lot of extra features that many won't need.

**Comment by wulforr** [comment]
@Zauberfisch  The code samples are present in the blog that you have mentioned, can you check once again

**Comment by Zauberfisch** [comment]
Hmm, that's weird. You are correct, I see them too now. Perhaps a temporary outage. I see they are github gists, but they are wrapped in iframes from usejounal.com.

Ok, I guess that invalidates the issue I had with the blog post. 

Though I'd actually like to raise the question if it wouldn't be benefitial to adopt the contents of this blog post into the official react documentation and expand on it further.
I still believe that an option to roll your own tool chain is much more important then the documentation gives attention to.

**Comment by wulforr** [comment]
I agree with you completely over this there should be info about setting your own react project from scratch in the official docs.
Maybe they will take a look at this.

**Comment by AnmolBansalDEV** [comment]
Is this still an issue? If so I'll like to work on this.
